server/controllers/events/testfixtures/test-repos/import-multiple-project/exp-output-import-dummy1.txt

```diff
@@ -12,5 +12,5 @@ The resources that were imported are shown above. These resources are now in
 your Terraform state and will henceforth be managed by Terraform.
 ```
 
-* :repeat: To **plan** this project again, comment:
+* :repeat: plan file was discarded, to **plan** this project again, comment:
```

Comment:
This seems like more of a summary statement. Generally we put summary statements on its own line between the triple backticks and the `* :repeat:`

Reply:
like this?

~~~
your Terraform state and will henceforth be managed by Terraform.
```

:put_litter_in_its_place: A plan file was discarded. Re-plan would be required before applying.

* :repeat: To **plan** this project again, comment:
  * `atlantis plan`
~~~